systest/scripts/install_lbaasv2.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -ex
+
+GLOBAL_ROUTED_MODE=False
+OS_CONTROLLER_IP=`/tools/bin/tlc --sid ${TEST_SESSION} symbols \
+    | grep openstack_controller1ip_data_direct \
+    | awk '{print $3}' | xargs`
+SSH_CMD="ssh -i /home/jenkins/.ssh/id_rsa_testlab -o StrictHostKeyChecking=no testlab@${OS_CONTROLLER_IP}"
+BIGIP_IP=`${SSH_CMD} "cat /home/testlab/ve_mgmt_ip"`
+BIGIP_IP=${BIGIP_IP%%[[:cntrl:]]}
+NETWORK_TYPES=${TEST_TENANT_NETWORK_TYPE}
+AGENT_LOC=git+https://github.com/F5Networks/f5-openstack-agent.git@${TEST_OPENSTACK_DISTRO}
```

Comment:
the TEST_OPENSTACK_DISTRO var here is being built with the SANITIZED_BRANCH var in the makefile. This works here, but won't work for newton or future versions because the branch names have changed to stable/<openstack_release>. We should use the BRANCH var: https://github.com/F5Networks/f5-openstack-lbaasv2-driver/blob/liberty/systest/Makefile#L24 as that is consistent across all of our repos.